ExternalRevisions/Vulkan-Tools_repo_revision

```diff
@@ -1 +1 @@
-2020cec4111c87d85b167d583180b839f0c736c5
+fb8f5a5d69f4590ff1f5ecacb5e3957b6d11daee
```

Comment:
Why the need for all the dependency updates, other than Vulkan-Headers?

No need to revert them if they're working, but I'm curious.

Reply:
No particular reason other than trying to keep all of them consistent with the same version to avoid issues.